style.css

```diff
@@ -186,30 +186,73 @@ hr {
 	right: 10px;
 }
 
-/*-------------Login Content Box----------------*/
+/*-------------Content Boxes----------------*/
+
```

Comment:
I have added this classes for the new additons for the pages. I have put them in a section of my css style sheet called 'Content Boxes' so I can find them easier.